day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,65 @@
+def in_words(int)
```

Comment:
Great job using a hash here Devin! This is exactly the kind of implementation we were hoping to see. If you have time to attempt the other challenges this weekend then I encourage it! You can time box the challenges to get practice with a time constraint, which you will see weekly at Turing! Nice work on the technical capstone Devin!